src/main/java/frc/robot/Constants.java

```diff
@@ -14,9 +14,9 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
-  public static final double SIMPLE_AUTON_SPEED = 0.2; // PLACEHOLDER
-  public static final double SIMPLE_AUTON_RUNTIME = 5.0; // PLACEHOLDER
-  public static final double MEDIUM_AUTON_OUTAKE_RUNTIME = 5.0; // PLACEHOLDER
+  public static final double SIMPLE_AUTON_SPEED = 0.5; // WEEK0
+  public static final double SIMPLE_AUTON_RUNTIME = 4.0; // WEEK0
+  public static final double MEDIUM_AUTON_OUTTAKE_RUNTIME = 2.0; // PLACEHOLDER
```

Comment:
Should be `// WEEK0`??

Reply:
I put that there to make it clearer what we've tested before and what we haven't
We'll still need to re-test some things but hopefully not runtime/speed/current?
I think that answers the question?